CHANGELOG.md

```diff
@@ -1,108 +1,91 @@
-| Bug   | Minicart missing quantity while switching views                                                                                         | [3571][]          |
-| Bug   | Order history data not being cleared from cache on logout                                                                               | [3520][]          |
-| Bug   | Pagebuilder image content-type: mobile image is visible on desktop                                                                      | [3515][]          |
-| Bug   | Pagebuilder links force browser to load full page                                                                                       | [3524][]          |
-| Bug   | Add to Cart on category page for non-default store ends up in 404 page if store view level unique catalog product suffixes are enabled. | [3553][]          |
-| Bug   | Category page fails to load intermittently when there is Apollo related console warning.                                                | [3575][]          |
-| Bug   | Dependency on Magento was sneaked in as part of the Computed Resolver work                                                              | [16][], [6][]     |
-| Bug   | \[bug]: When clicking a submenu in the mega the submenu won't close                                                                     | [3599][]          |
-| Bug   | Fix console warning for currency                                                                                                        | [3622][]          |
-| Bug   | Remove server chunk generation from webpack                                                                                             | [3608][]          |
-| Bug   | \[Issue] Improve Gallery/AddToCartButton.js                                                                                             | [3573][]          |
-| Bug   | \[bug]: Breadcrumbs 'Shop' link redirects to '/shopnull'                                                                                | [3603][]          |
-| Bug   | Newsletter form displays two errors                                                                                                     | [3584][]          |
-| Bug   | Home page tabbing leads to page transition                                                                                              | [3625][]          |
-| Bug   | Browser Reload on any Category load all products from default category                                                                  | [3655][], [18][]  |
+
+## Highlights  
+  
+-   Added shimmer loaders on dynamic blocks to avoid layout shift. Shimmer loaders will only be rendered if minHeight is specified on the row containing a dynamic block. This gives flexibility to users to display a shimmer or not (if a dynamic block would not render most of the time, we probably do not want to display a shimmer) — [3713][]
```

Comment:
```suggestion
-   Added shimmer loaders for Dynamic blocks to avoid layout shift. Shimmer loaders are only rendered if `minHeight` is specified on the Row containing the Dynamic block. This implementation gives you a choice: You can set the Row's `minHeight` to render the shimmer or remove the `minHeight` to remove the shimmer. For Dynamic blocks that rarely render, we recommend removing the `minHeight` property from the Rows containing those Dynamic blocks. — [3713][]
```